### DISCUSSION TITLE
Saving recordings to Digital Ocean Spaces

### DISCUSSION THREAD

**Discussion by Mohit-3196** [ant-media/Ant-Media-Server#5503] [General]
In recent times, I have come across some queries about MP4 recordings not getting uploaded to Digital Ocean Spaces.
Upon checking the reason this issue is happening is the Endpoint.
Unlike AWS S3 and other S3 compatible buckets, for Digital Ocean Spaces, there's no requirement for **Bucket name** in the **Endpoint** field.

For instance:
If the region is: sfo3
and, bucket-name is: ams-bucket
The endpoint will be: https://sfo3.digitaloceanspaces.com/

I hope it helps.
Thank you.

**Comment by dmtan90** [comment]
@Mohit-3196 you can follow this user guide https://antmedia.io/how-to-record-streams-to-s3-compatible-systems/

**Reply by Mohit-3196** [reply]
Thank you @dmtan90 ,
Much appreciated.
